### PR TITLE
A few small changes to augmentation in recognition training

### PR DESCRIPTION
**Pull request by notiho**
- refactored augmentation code so that all of it is in one place
- changed parameters of `ElasticTransform` to something more reasonable
- added pixel dropout
- fixed random seed in validation data loader so that augmentation in validation is deterministic

### CODE REVIEW

File: kraken/lib/train.py [hunk 1]
```diff
@@ -574,12 +603,16 @@ def train_dataloader(self):
                           collate_fn=collate_sequences)
 
     def val_dataloader(self):
+        generator = torch.Generator()
```
Comment (mittagessen) [inline kraken/lib/train.py:606]: This isn't necessary as the module doesn't do any validation time augmentation (and for CLI use at least lightning's `seed_everything` also generates fixed seeds for workers.

Comment (notiho) [reply kraken/lib/train.py:606]: For my attempt to disable augmentation for validation, see 8fadba9. This does however not work if training and validation share the same dataset instance.

**Comment by notiho** [comment]
I now also removed two augmentation steps (90° rotation and flip) from the segmentation augmentation, which also seem a bit excessive.

File: kraken/lib/train.py (hunk 1 above)
Comment (mittagessen) [reply kraken/lib/train.py:608]: Ah sorry, I mixed up the trainers (segmentation doesn't use validation augmentation). In any case, as we're already using pytorch-lightning `seed_everything` is really the way to go here.

Comment (notiho) [reply kraken/lib/train.py:612]: Sure, just changed it. I noticed that `seed_everything` will always log something at info level, so I silenced that for that call. But idk if I have the most elegant way to do that.
